src/Query/AzureServiceBus/AzureResourceId.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Particular.ThroughputQuery.AzureServiceBus;
+
+using System;
+
+public static class AzureResourceId
```

Comment:
Have you considered using https://learn.microsoft.com/en-us/dotnet/api/azure.core.resourceidentifier?view=azure-dotnet. Since we have a reference to Azure.Core we should have that type available?

Reply:
I'll see if I can use that. That would save some maintenance on our part.

Reply:
Thanks, @danielmarbach. That has everything needed, and it was a simple swap.